src/installer.ts

```diff
@@ -42,31 +39,62 @@ export async function getNode(
     console.log(`Found in cache @ ${toolPath}`);
   } else {
     console.log(`Attempting to download ${versionSpec}...`);
-    let info = await getInfoFromManifest(versionSpec, stable, token);
-    if (!info) {
-      console.log(
-        'Not found in manifest.  Falling back to download directly from Node'
-      );
-      info = await getInfoFromDist(versionSpec);
-    }
-
-    if (!info) {
-      throw new Error(
-        `Unable to find Node version '${versionSpec}' for platform ${osPlat} and architecture ${osArch}.`
-      );
-    }
-
-    console.log(`Acquiring ${info.resolvedVersion} from ${info.downloadUrl}`);
-
     let downloadPath = '';
+    let info: INodeVersionInfo | null = null;
+
+    //
+    // Try download from internal distribution (popular versions only)
+    //
     try {
-      downloadPath = await tc.downloadTool(info.downloadUrl, undefined, token);
+      info = await getInfoFromManifest(versionSpec, stable, token);
+      if (info) {
+        console.log(
+          `Acquiring ${info.resolvedVersion} from ${info.downloadUrl}`
+        );
+        downloadPath = await tc.downloadTool(
+          info.downloadUrl,
+          undefined,
+          token
+        );
+      } else {
+        console.log(
+          'Not found in manifest.  Falling back to download directly from Node'
+        );
+      }
     } catch (err) {
-      if (err instanceof tc.HTTPError && err.httpStatusCode == 404) {
-        return await acquireNodeFromFallbackLocation(info.resolvedVersion);
+      // Rate limited?
```

Comment:
403 indicates rate limited. Either way print to Info and fallback.

Reply:
I thought a 429 would be rate limiting but I guess that's usually - hey back off where 403 is you hit the limit.  I guess a bad token would be 401

Reply:
:shrug: i implemented based on the docs for [GitHub rate limit rules](https://developer.github.com/v3/#rate-limiting) (403). But you are right, based on some light searching 429 may be more common rate limit error code. I can update the code to handle both. At most will improve the error message; will still fallback either way.